mlflow/pytorch/__init__.py

```diff
@@ -437,10 +557,21 @@ class LinearNNModel(torch.nn.Module):
         predict X: 12.0, y_pred: 23.96
         predict X: 30.0, y_pred: 60.13
     """
+
     import torch
 
     pickle_module = pickle_module or mlflow_pytorch_pickle_module
 
+    if isinstance(pytorch_model, OrderedDict):
+        save_state_dict(
+            state_dict=pytorch_model,
+            path=path,
+            mlflow_model=mlflow_model,
+            pickle_module=pickle_module,
+            **kwargs,
+        )
+        return
+
```

Comment:
Is this for allowing `save_model` to log `state_dict`?

Reply:
yes @harupy